HousingRepairsOnline.AddressIngestion/Services/InsertAddressesToCosmosDb.cs

```diff
@@ -25,19 +25,19 @@ public async Task Execute(IEnumerable<PropertyAddress> propertyAddresses)
                 if (propertyAddress.AddressLine.FirstOrDefault() is "" or null)
                 {
                     this.logger.LogInformation($"AddressLine for property {propertyAddress.Reference.ID} is null or empty");
+                    continue;
```

Comment:
The first of these validation checks to fails will invoke a `continue` which will exit that loop iteration and proceed to the next iteration.
This would not highlight (via logging) other issues with the same address entity.
Each of the validation checks should be performed prior to exiting the iteration.